packages/mobile/src/account/selectors.test.ts

```diff
@@ -30,7 +30,38 @@ describe('selectors', () => {
         },
       }
 
-      expect(getPaymentRequests(state)).toEqual([goodRequest])
+      expect(getIncomingPaymentRequests(state)).toEqual([goodRequest])
+    })
+  })
+
+  describe(getOutgoingPaymentRequests, () => {
```

Comment:
I appreciate the thoroughness and I'm not sure why wrote the existing test above but IMHO selectors and reducers are too simple to be worth testing. Definitely don't remove this now that it's done but for the future I wouldn't bother with this.